db/migrations/20220803110900-add-config-columns.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const tableNames = ["RepoSyncStates"];
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await Promise.all([
```

Comment:
Yeah, you're right, this could be simplified. This code created the `config` field to 3 different tables before, so I just removed the 2 we don't need for now. I don't really want to touch the database migration code again because it's not covered with tests and bad things happen if I screw it up.

Reply:
why remove the other 2 tables?

Reply:
Because we're not using them, yet. We'd need to build a whole mechanism around when to store config on the level of an installation or subscription. How should users contribute an installation-level or subscription-level config? That needs more thinking.